zygrader/ui/layers.py

```diff
@@ -316,6 +316,20 @@ def build(self):
             prompt += [f"[Invalid {TYPE_STR}]"]
         self.component.set_message(prompt)
 
+    def __autocomplete(self):
+        path = os.path.expanduser(self.component.text)
+        directory, partial_name = os.path.split(path)
```

Comment:
There is a crash in autocomplete if the path is invalid

```suggestion
        directory, partial_name = os.path.split(path)
        if not os.path.exists(directory):
            return
```